Library/Formula/vobcopy.rb

```diff
@@ -14,6 +11,11 @@ def install
                              "--with-lfs",
                              "--with-dvdread-libs=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/vobcopy -h 2>&1", 1)
```

Comment:
The most realistic test would be to have the user keep a movie DVD in their DVD drive. Saving that, we'd have to create a VOB file from a DVD. I don't know if you want a test file that big in the test fixtures...